### PR TITLE
Stale option

### PR DESCRIPTION
**Pull request by rbanffy**
## Pull Request Checklist

**Is this in reference to an existing issue?**

No.

#### General

- [X] Update Changelog following the conventions laid out on [Our CHANGELOG Guidelines ](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)

- [X] Update README with any necessary configuration snippets

- [X] Binstubs are created if needed

- [X] RuboCop passes

- [X] Existing tests pass

#### New Plugins

- [N/A] Tests

- [N/A] Add the plugin to the README

- [N/A] Does it have a complete header as outlined [here](http://sensu-plugins.io/docs/developer_guidelines.html#coding-style)

#### Purpose

Issue a warning if the number of stale checks go over a specified count.

#### Known Compatibility Issues

None.

### CODE REVIEW

**Review by luisdavim — APPROVED** [review]
LGTM

**Comment by rbanffy** [comment]
Reduced the number of commits by changing the past.

File: bin/check-aggregate.rb [hunk 1]
```diff
@@ -130,6 +130,16 @@ class CheckAggregate < Sensu::Plugin::Check::CLI
          boolean: true,
          default: false
 
+  option :stale,
```
Comment (majormoses) [inline bin/check-aggregate.rb]: if we are going to use `--stale-count` then I think this should be `--stale-percentage` as looking at the 2 options before reading the description I assumed it was any stale seeing both `--stale` and `--stale-count`

**Review by majormoses — CHANGES_REQUESTED** [review]
Overall this looks good, the thing I think we can improve here is the name of the options.

**Review by majormoses — APPROVED** [review]
LGTM

**Comment by majormoses** [comment]
released: https://rubygems.org/gems/sensu-plugins-sensu/versions/2.2.0

**Pull request merged**